### PR TITLE
Adding user agent.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

File: src/main/scala/com.gu.contentapi.client/GuardianContentClient.scala [hunk 1]
```diff
@@ -4,6 +4,7 @@ import com.gu.contentapi.client.model._
 import com.gu.contentapi.client.parser.JsonParser
 import com.gu.contentapi.client.utils.QueryStringParams
 import dispatch.{FunctionHandler, Http}
+import org.json4s.BuildInfo
```
Comment (cb372) [inline src/main/scala/com.gu.contentapi.client/GuardianContentClient.scala]: Wrong BuildInfo!

Comment (LATaylor-guardian) [inline src/main/scala/com.gu.contentapi.client/GuardianContentClient.scala]: Never trust IntelliJ!

Comment (mchv) [inline src/main/scala/com.gu.contentapi.client/GuardianContentClient.scala]: > Never trust IntelliJ!
I told you so :smile:

File: project/build.scala [hunk 2]
```diff
@@ -14,6 +17,11 @@ object ContentApiClientBuild extends Build {
     id = "content-api-client",
     base = file(".")
   )
+  .enablePlugins(BuildInfoPlugin)
+  .settings(
+      buildInfoKeys := Seq[BuildInfoKey](version),
+      buildInfoPackage := "com.gu.contentapi.buildInfo"
```
Comment (cb372) [inline project/build.scala]: I don't really like the camel-cased package name

**Comment by cb372** [comment]
You'll need to update this line: https://github.com/guardian/content-api-scala-client/blob/master/src/main/scala/com.gu.contentapi.client/GuardianContentClient.scala#L43

**Comment by cb372** [comment]
:+1:

**Pull request merged**